R/as.vp.R

```diff
@@ -0,0 +1,19 @@
+#' Convert a dataframe into a vp object
+#'
+#' @param data a dataframe created from a VPTS CSV file
+#' @returns a bioRad vp object
+#' @examples
+#' # load vp data as a data.frame:
+#' df <- as.data.frame(example_vp)
+#' # convert the data.frame to a vp object:
+#' as.vp(df)
+#' @export
+as.vp <- function(data) {
+  assertthat::assert_that(inherits(data,"data.frame"))
+
+  vpts <- as.vpts(data)
+  
+  assertthat::assert_that(length(vpts$datetime) == 1, msg="multiple timestamps found, data is not a single vertical profile")
```

Comment:
You might want to extent this check to if height bins are duplicated. Also do height bins need to be regularly spaced?

Reply:
I believe those checks are performed already by as.vpts() which is used under the hood, but I'll double check

Reply:
Rows can be removed, so no check for regular spacing currently:
> df=as.data.frame(example_vp, suntime=F)
> as.vp(df[-5,])

Reply:
related to https://github.com/adokter/bioRad/issues/697, which requires a fix before next release